### PR TITLE
A: https://oploverz.co.in/

### PR DESCRIPTION
**Pull request by eporubin**
Hi, could you please review this adservers found on this page. 
Kind regards.

### CODE REVIEW

File: src/advert/adservers.txt [hunk 1]
```diff
@@ -48,6 +48,7 @@
 ||filebanner.com^$third-party
 ||forumiklan.com^$third-party
 ||goingsilltrusty.com^
+||googleusercontent.com^
```
Comment (gmarcher) [inline src/advert/adservers.txt:51]: This domain serves general images, not ad specific, if you want to block it must be specific

**Comment by gmarcher** [comment]
are you using Google Chrome+adblock plus?

File: src/advert/adservers.txt (hunk 1 above)
Comment (eporubin) [reply src/advert/adservers.txt:51]: Yes the image hovering on top of the page comes from it so I assumed  its the problem. But thanks for letting me know. 
To make it specific 
`15.235.11.45,oploverz.co.in##a[title*="388hero slot gacor"]`
would do the same but through Element Hiding

**Comment by eporubin** [comment]
> 

Yes I am testing on Google Chrome +adblock plus (with ABPindo+easyList enabled) and also on the phone with  Samsung Internet + AdBlockPlus for Sumsung internet (Bahasa Indonesia, Malayu + English list enabled)

**Review by gmarcher — CHANGES_REQUESTED** [review]

**Pull request merged**